bundler/helpers/v1/monkey_patches/definition_ruby_version_patch.rb

```diff
@@ -11,7 +11,7 @@ def index
           Gem::Specification.new("ruby\0", requested_version)
       end
 
-      %w(2.5.3p105 2.6.10p210 2.7.6p219 3.0.4p208).each do |version|
+      %w(2.5.3p105 2.6.10p210 2.7.6p219 3.0.4p208, 3.1.2p20).each do |version|
```

Comment:
You don't need commas with `%w` and in fact this would unintentionally add a version named `3.0.4p208,` to the array

```suggestion
      %w(2.5.3p105 2.6.10p210 2.7.6p219 3.0.4p208 3.1.2p20).each do |version|
```

Reply:
But as per https://github.com/dependabot/dependabot-core/pull/6257#discussion_r1043099175, I don't think this update is necessary, so the better suggestion would be

```suggestion
      %w(2.5.3p105 2.6.10p210 2.7.6p219 3.0.4p208).each do |version|
```

Reply:
Thanks @deivid-rodriguez much obliged!